build.gradle.kts

```
@@ -43,7 +43,8 @@ subprojects {
       languageVersion = "1.3"
     }
   }
-  extensions.findByType<KotlinProjectExtension>()?.apply {
-    explicitApiWarning() // TODO(egor): Change to explicitApi() and fix errors.
+  afterEvaluate {
+    // TODO(egor): Change to explicitApi() and fix errors.
+    the<KotlinProjectExtension>().explicitApiWarning()
```

rather than this, you can wait until the plugin is applied

```suggestion
  pluginManager.withPlugin("org.jetbrains.kotlin.jvm") {
    // TODO(egor): Change to explicitApi() and fix errors.
    configure<KotlinProjectExtension> {
      explicitApiWarning()
    }
```

---

Thanks!